src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1967,6 +1967,12 @@ public static IEnumerable<DValue<RecordValue>> StandardTableNodeRecords(IRContex
                 else if (!forceSingleColumn && arg is BlankValue bv && tableType.FieldNames.Count() > 1)
                 {
                     return DValue<RecordValue>.Of(bv);
+                }
+
+                // Handle the Unknown return type functions: Remove, Coelesce

```

Comment:
>Handle the Unknown return type functions [](http://example.com/codeflow?start=19&length=40)

It's not really an unknown return type - we do know the type. It's mostly untyped blank values.